### PR TITLE
[Feature]:  Cert-Manager Dependency

### PR DESCRIPTION
**Pull request by sadath-12**
Added Cert-Manager Dependency and CRD for issue https://github.com/OT-CONTAINER-KIT/helm-charts/issues/129

### CODE REVIEW

File: charts/redis-operator/Chart.yaml [hunk 1]
```diff
@@ -19,3 +19,10 @@ keywords:
 - opstree
 - kubernetes
 - openshift
+
+dependencies:
+  - name: cert-manager
+    version: v1.12.0
+    repository: https://charts.jetstack.io
+    alias: cert-manager
+    # condition: cert-manager.enabled
```
Comment (shubham-cmyk) [inline charts/redis-operator/Chart.yaml:28]: User should choose to install or not

File: charts/redis-operator/Chart.yaml [hunk 2]
```diff
@@ -19,3 +19,10 @@ keywords:
 - opstree
 - kubernetes
 - openshift
+
+dependencies:
```
Comment (shubham-cmyk) [inline charts/redis-operator/Chart.yaml:23]: Can we use `import-values: # (optional)`  this field to install the crd

**Comment by sadath-12** [comment]
sure , what should be the default ?

File: charts/redis-operator/Chart.yaml (hunk 2 above)
Comment (shubham-cmyk) [reply charts/redis-operator/Chart.yaml:23]: Simply add 

```
cert-manager:
  installCRDs: true
```

To the values.yaml and remove the crd

**Comment by sadath-12** [comment]
or how about adding a boolean value in values.yaml and conditioning the crd yaml

**Comment by shubham-cmyk** [comment]
> sure , what should be the default ?

Make the default to be `true`

**Comment by shubham-cmyk** [comment]
> or how about adding a boolean value in values.yaml and conditioning the crd yaml

Could work. 

Add condition & add crd enable in values.yaml

**Comment by sadath-12** [comment]
> > or how about adding a boolean value in values.yaml and conditioning the crd yaml
> 
> Could work.
> 
> Add condition & add crd enable in values.yaml

sure

File: charts/mongodb-operator/values.yaml
Comment (shubham-cmyk) [inline charts/mongodb-operator/values.yaml:1]: Remove changes from here

File: charts/fluentd/templates/custom-config.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {{- if .Values.customConfiguration }}
 ---
-apiVersion: v1
```
Comment (shubham-cmyk) [inline charts/fluentd/templates/custom-config.yaml:3]: Remove this change

**Pull request merged**